bolt-design-system/bolt
# Modal | JS | Visible shift of content when opening/closing video modal

## Jira

- http://vjira2:8080/browse/BDS-1681

## Summary

Prevent visible shift of content when modal opens and closes.

## Details

Check if window has a scrollbar, if so get its width, and add/remove that width in padding to the page BODY on modal show/hide.

## How to test

Run locally in Chrome, FF, IE. Test with scrollbars set to "always show" and "auto". Make sure the background page content does not shift when modal opens/closes.
